pkg/emb6/Makefile

```diff
@@ -0,0 +1,33 @@
+PKG_NAME=emb6
+PKG_URL=git://github.com/hso-esk/emb6.git
```

Comment:
Any particular reason for not using the http URL here?

(Just realized that in some networks other ports may be blocked, while port 80 is usually open.)